### PR TITLE
chore: adding Nebula as GH owners [NEBULA-1290]

### PR DESCRIPTION
**Pull request by metju90**
Adding team Nebula as code owners for files that could break local-engine

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,5 @@
 * @snyk/hammerhead
 src/snyk/common/services/learnService.ts @snyk/owl
 src/test/unit/common/services/learnService.test.ts @snyk/owl
+src/snyk/snykCode/codeSettings.ts @snyk/nebula
+src/test/unit/snykCode/codeSettings.test.ts @snyk/nebula
```
Comment (metju90) [inline .github/CODEOWNERS:5]: `@snyk/owl` does not seem to exist anymore

**Review by bastiandoetsch — CHANGES_REQUESTED** [review]
Hi @metju90 , I'll discuss with the team, if we want code-ownership of the codeSettings in VSCode with Nebula.

**Comment by metju90** [comment]
> Hi @metju90 , I'll discuss with the team, if we want code-ownership of the codeSettings in VSCode with Nebula.

@bastiandoetsch shared ownership will be best. We want to be notified of potentially breaking changes for local-engine.

**Comment by metju90** [comment]
@bastiandoetsch Code files are now co-owned between hammerhead, zenith and nebula

**Review by bastiandoetsch — APPROVED** [review]

**Pull request merged**